src/ucp/wireup/wireup.c

```diff
@@ -633,15 +675,23 @@ void ucp_wireup_process_ack(ucp_worker_h worker, const ucp_wireup_msg_t *msg)
 
     ucs_assert(ep->flags & UCP_EP_FLAG_REMOTE_ID);
     ucs_assert(ep->flags & UCP_EP_FLAG_CONNECT_REP_SENT);
-    ucs_assert(ep->flags & UCP_EP_FLAG_LOCAL_CONNECTED);
 
+    if (!(ep->flags & UCP_EP_FLAG_LOCAL_CONNECTED)) {
+        ucs_assert(ucp_ep_has_cm_lane(ep) &&
+                   (ep->flags & UCP_EP_FLAG_DISCONNECTED_CM_LANE));
+        return;
```

Comment:
pls add a comment that we drop the msg since close protocol or err handling is started